themes/custom/droopler_theme/scss/layout/_global.scss

```diff
@@ -19,12 +19,6 @@
   main.main-content {
     width: 100%;
   }
-  h1 {
-    @include font-main-header-big;
```

Comment:
Czy te mixiny są do wyrzucenia, czy może są używane w innych miejscach? Czy to zmienia wygląd paragrafów?